test-utils/src/monerod.rs

```diff
@@ -34,7 +32,7 @@ const MONEROD_SHUTDOWN_TEXT: &str = "Stopping cryptonote protocol";
 ///
 /// This function will set `regtest` and the P2P/ RPC ports so these can't be included in the flags.
 pub async fn monerod<T: AsRef<OsStr>>(flags: impl IntoIterator<Item = T>) -> SpawnedMoneroD {
-    let path_to_monerod = download::check_download_monerod().await.unwrap();
+    let path_to_monerod = find_root().join("monerod");
```

Comment:
This works with windows? `monerod.exe` not needed?

Reply:
yep, maybe this will break but for now it works

Reply:
I'll clear the cache see if it still works...